src/applications/vaos/new-appointment/redux/helpers/getReasonCode.js

```diff
@@ -16,9 +16,10 @@ export function getReasonCode({ data, isCC, isAcheron, isDS }) {
   const apptReasonCode = PURPOSE_TEXT_V2.find(
     purpose => purpose.id === data.reasonForAppointment,
   )?.commentShort;
+  // code is utilized when Acheron is off; need to cleanup
   const code = PURPOSE_TEXT_V2.filter(purpose => purpose.id !== 'other').find(
```

Comment:
Did this comment get removed intentionally?

Reply:
This comment is added as reminder when we do cleanup for Acheron toggle flag removal.